test/app.spec.js

```diff
@@ -46,5 +46,17 @@ describe('app', function () {
     expect(await app.collaborate('collaboration name', 'fake', collaborationOptions)).to.equal(collaboration)
   })
 
+  it('can get concurrently created collaboration shared', async () => {
```

Comment:
I added this test to my current master (without your changes to `src/app.js`) and the test passes fine.